pyEpiabm/pyEpiabm/tests/test_unit/test_routine/test_file_population_config.py

```diff
@@ -179,23 +190,34 @@ def test_print_population_loc(self, mock_write, mock_read):
         mock_write.assert_called_once()
         self.assertEqual(mock_write.call_args[0], ('output.csv',))
 
+    @patch('logging.info')
     @patch("pandas.read_csv")
     @patch("copy.copy")
-    def test_print_population(self, mock_copy, mock_read):
+    def test_print_population(self, mock_copy, mock_read, mock_house_log):
```

Comment:
Do we check that the mock_house_log has been called? I missed this if so, but would be good to check

Reply:
This shouldn't have been left in - I have removed.